api-gateway/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM hirokimatsumoto/alpine-openjdk-11
+
+RUN apk update && apk add bash
+
+VOLUME /tmp
+ADD ./target/api-gateway-0.0.1-SNAPSHOT.jar api-gateway.jar
```

Comment:
let's unlink in all dockerfiles from specific versions and replace them by regex search pattern.